ExtraDry/ExtraDry.Blazor/Components/Standard/Suspense.razor.cs

```diff
@@ -51,9 +51,13 @@ public partial class Suspense<TModel> : ComponentBase, IExtraDryComponent {
     [Parameter(CaptureUnmatchedValues = true)]
     public Dictionary<string, object>? UnmatchedAttributes { get; set; }
 
-    /// <inheritdoc cref="IndicatorSize" />
+    /// <inheritdoc cref="Blazor.IndicatorSize" />
     [Parameter]
-    public IndicatorSize Size { get; set; } = IndicatorSize.Standard;
+    public IndicatorSize IndicatorSize { get; set; } = IndicatorSize.Standard;
```

Comment:
This concept of IndicatorSize versus SpinnerSize I find a bit confusing.  Shouldn't they have consistent sizes?  E.g. we'd never have a small spinner and a large indicator, would we?  Is this just to enable not showing the spinner?  Perhaps having a single size and then separate mechanism for what is shown.